[BUG] test_re_replace_no_unicode_fallback test failes on integratoin tests Yarn

**Describe the bug**
test_re_replace_no_unicode_fallback integration test failed on yarn cluster

[2023-04-04T18:17:46.871Z] E                   py4j.protocol.Py4JJavaError: An error occurred while calling z:org.apache.****.sql.rapids.ExecutionPlanCaptureCallback.assertDidFallBack.
[2023-04-04T18:17:46.871Z] E                   : java.lang.AssertionError: assertion failed: Could not find RegExpReplace in the GPU plans:
[2023-04-04T18:17:46.871Z] E                   GpuColumnarToRow false
[2023-04-04T18:17:46.871Z] E                   +- GpuProject [gpuregexpreplace(a#12, TEST, PROD) AS regexp_replace(a, TEST, PROD, 1)#14], true
[2023-04-04T18:17:46.871Z] E                      +- GpuRowToColumnar targetsize(2147483647)
[2023-04-04T18:17:46.871Z] E                         +- *(1) Scan ExistingRDD[a#12]
[2023-04-04T18:17:46.871Z] E

--- a/integration_tests/src/main/python/regexp_no_unicode_test.py
+++ b/integration_tests/src/main/python/regexp_no_unicode_test.py
@@ -35,7 +35,7 @@ def test_rlike_no_unicode_fallback():
     gen = mk_str_gen('[abcd]{1,3}')
     assert_gpu_fallback_collect(
         lambda spark: unary_op_df(spark, gen).selectExpr(
-            'a rlike "ab"'),
+            'a rlike "ab+"'),
         'RLike',
         conf=_regexp_conf)
 
@@ -44,7 +44,7 @@ def test_re_replace_no_unicode_fallback():
     gen = mk_str_gen('.{0,5}TEST[\ud720 A]{0,5}')
     assert_gpu_fallback_collect(
         lambda spark: unary_op_df(spark, gen).selectExpr(
-            'REGEXP_REPLACE(a, "TEST", "PROD")'),
+            'REGEXP_REPLACE(a, "[A-Z]+", "PROD")'),
         'RegExpReplace',
         conf=_regexp_conf)
 

